client/my-sites/plans/navigation.jsx

```diff
@@ -21,6 +21,8 @@ import { isATEnabled } from 'lib/automated-transfer';
 import isSiteOnFreePlan from 'state/selectors/is-site-on-free-plan';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { getSite, isJetpackSite } from 'state/sites/selectors';
+import { SIDEBAR_SECTION_MANAGE } from '../sidebar/constants';
+import { expandMySitesSidebarSection as expandSection } from 'state/my-sites/sidebar/actions';
```

Comment:
Aliasing is sometimes used to avoid shadowing variables, but in this case, we needed the verbosity for what this action does in a global context, and likely most folks found it to be too long to type 🤷‍♀

Reply:
>  likely most folks found it to be too long to type 
Who type the whole function name nowadays? 😁

I was just pointing out that this function is aliased all the time... Which seems pretty odd.

Reply:
Might be a copy paste effect too, you don't need to alias if you don't want to.

Reply:
I prefer to keep the same pattern found in the other places. 

Someone down the line might just look for `expandSection` across the project and miss my call for example...

Reply:
It's fine either way as long as folks don't obsfucate `expandMySitesSidebarSection` with something like a string concat or other types of meta programing. We should be able to do a simple string search for usages in either form.